couchdb/templates/statefulset.yaml

```diff
@@ -125,6 +129,40 @@ spec:
 {{- if .Values.sidecars }}
 {{ toYaml .Values.sidecars | indent 8}}
 {{- end }}
+{{- if .Values.telemetry.enabled }}
+        - name: telemetry
+          image: "{{ .Values.telemetryImage.repository }}:{{ .Values.telemetryImage.tag }}"
+          imagePullPolicy: {{ .Values.telemetryImage.pullPolicy }}
+          ports:
+            - name: telemetry
+              containerPort: 9984
+          args: ["--couchdb.uri=http://127.0.0.1:5984"]
+          livenessProbe:
+            httpGet:
+              path: /metrics
+              port: 9984
+          readinessProbe:
+            httpGet:
+              path: /metrics
+              port: 9984
+          resources:
+{{ toYaml .Values.telemetry.resources | indent 12 }}
+          env:
+            - name: DATABASES
+              value: {{ .Values.telemetry.databases }}
+{{- if not .Values.allowAdminParty }}
+            - name: "COUCHDB.USERNAME"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ template "couchdb.defaultsecret" . }}
```

Comment:
Now I think this one should be back to `couchdb.fullname`, but please keep the `couchdb.defaultsecret` below ;)

Reply:
looking at it again, I think the original was correct. The admin username and password are stored [here](https://github.com/apache/couchdb-helm/blob/master/couchdb/templates/secrets.yaml#L5) so they should both refer to `{{ template "couchdb.fullname" . }}`

Reply:
Ah, I'm very sorry. Looking at https://github.com/apache/couchdb-helm/blob/master/couchdb/templates/secrets.yaml#L12 I can see where the actual values come from.